capstone/pom.xml

```diff
@@ -116,7 +116,7 @@
                 <failOnMissingWebXml>false</failOnMissingWebXml>
                 <webResources>
                     <resource>
-                        <directory>${basedir}/target/angular/data-demo/dist/data-demo</directory>
+                        <directory>${basedir}/target/angular/data_demo/dist/data-demo</directory>
```

Comment:
I thought we had already fixed all of this. Is this still the case @HEAD?

Reply:
I keep getting errors after the pom.xml conflict we had the other time. This fixed it.

Reply:
sgtm.